app/views/groups/_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= simple_form_for [@draw, @group] do |f| %>
```

Comment:
not 100% sure, simple_form figures it out I think. (see this [stackoverflow](http://stackoverflow.com/questions/23945600/rails-4-action-routes-with-simple-form-and-shallow-nested-resources))